src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,8 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            internal FixedString.Size260 cFileName;
+            internal FixedString.Size14 cAlternateFileName;
```

Comment:
> I think it's way more error prone to have this custom struct thing that's very non-standard and difficult for someone who stumbles upon it in the code to understand how it all works.

I don't agree. You have to either write tons of wrappers and/or rely on everyone using the constants correctly (let alone not messing it up in your own struct). The fact that you need to use `fixed() {}` blocks to even access these makes accessing this code horrible. If you can show me a solution that doesn't require `fixed` blocks and manual connecting of buffer size constants I'd be happy to entertain it.

I also don't buy that looking at well-documented struct inside a P/Invoke struct is that onerous for a well-versed C# programmer.